### ISSUE TITLE
7.4 Issue With Array Key Exist

### ISSUE THREAD

**Issue by ProdigyView** [stripe/stripe-php#814]
Found this while running unit tests:

```
array_key_exists(): Using array_key_exists() on objects is deprecated. Use isset() or property_exists() instead

/code/vendor/stripe/stripe-php/lib/ApiRequestor.php:402
/code/vendor/stripe/stripe-php/lib/ApiRequestor.php:125
/code/vendor/stripe/stripe-php/lib/ApiOperations/Request.php:57
/code/vendor/stripe/stripe-php/lib/ApiOperations/Create.php:23
```

The code that produces the bug is:

```
if (array_key_exists('request-id', $rheaders)) {
            self::$requestTelemetry = new RequestTelemetry(
                $rheaders['request-id'],
                Util\Util::currentTimeMillis() - $requestStartMs
            );
        }
```
I;m unsure if $rheaders should be an object or array.

**Comment by ob-stripe** [comment]
@ProdigyView Thanks for the report! This was already fixed in #725 and released in stripe-php 7.0.0.

**Comment by fabswt** [comment]
FYI, getting this error in v6.43.1:

> array_key_exists(): Using array_key_exists() on objects is deprecated. Use isset() or property_exists() instead
> File: /var/www/html/clickandspeak/vendor/stripe/stripe-php/lib/ApiRequestor.php
> Line: 402

Was not getting the error with v5.9.2.  
OK in 7.28.0.
